tests/cli/workflow/balsamic/test_cli_balsamic_config_case.py

```diff
@@ -262,28 +262,6 @@ def test_error_normal_only(
     # WHEN dry running
     result = cli_runner.invoke(config_case, [case_id, "--dry-run"], obj=balsamic_context)
 
-    # THEN a start case error should be raised
-    assert result.exit_code != EXIT_SUCCESS
```

Comment:
Should you not assert some kind of state? Should the test be a succes or should it fail?

Reply:
It's there at the bottom. GitHub had a weird way displaying the changes since I removed an outdated test
```
    # THEN command should be generated successfully
    assert result.exit_code == EXIT_SUCCESS
```